ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -565,6 +580,7 @@ PrjFS_Result PrjFS_DeleteFile(
     struct stat path_stat;
     if (0 != stat(fullPath, &path_stat))
     {
+        LogWarning("PrjFS_DeleteFile: stat call failed fullPath=%s errno=%d", fullPath, errno);
```

Comment:
As a convenience, we might also want to log `strerror(errno)` where we log `errno` so we don't have to manually look up numeric error constants.

Reply:
This is an exellent point!  I've tried to update all places that could use this.  Please let me know if I missed one.